apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/features/BulkApiLiveTest.java

```diff
@@ -69,7 +69,7 @@ public void testExtractArchive() throws Exception {
                                                      .extractArchive(containerName, payload, "tar.gz");
          assertEquals(extractResponse.getCreated(), OBJECT_COUNT);
          assertTrue(extractResponse.getErrors().isEmpty());
-         assertEquals(api.getContainerApi(regionId).get(containerName).getObjectCount(), OBJECT_COUNT);
+         assertEquals(api.getContainerApi(regionId).get(containerName).getObjectCount(), Long.valueOf(OBJECT_COUNT));
```

Comment:
I´m not completely sure about what you mean. At the very beginning I went for the easy way and kept the object count as long, and make it 0 when null. nacx was probably right saying that would not be correct so I made it Long. 
In the tests, sometimes constant OBJECT_COUNT is compared with extractResponse.getCreated() , and therefore is a long, and sometimes it compared with getObjectCount() and therefore we must convert it to Long. 
I wonder if your concern is about object count long vs Long , or about how we deal with constants in tests, as they sometimes are compared with long and with Long. 
Thanks again.